app/src/main/java/com/example/reflector_android/Models/BlogRecyclerAdapter.kt

```diff
@@ -1,53 +1,66 @@
 package com.example.reflector_android.Models
 
-import android.os.Build
+import android.content.Intent
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import androidx.annotation.RequiresApi
 import androidx.recyclerview.widget.RecyclerView
 import com.example.reflector_android.R
 import com.example.reflector_android.network.Article
 import kotlinx.android.synthetic.main.article_item.view.*
 
-class BlogRecyclerAdapter(val articles: MutableList<Article>?) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+class BlogRecyclerAdapter(val articles: MutableList<Article>?) : RecyclerView.Adapter<CustomViewHolder>() {
```

Comment:
I think BlogRecyclerAdapter should move out of the "Models" folder and into a new directory. This is because BlogRecyclerAdapter isn't really a data model. It's more like a Controller that controls what happens in a view. Maybe make a directory called "ArticleList" to put all your Adapter and ViewHolder stuff in that relates to the Article List?